dbt/models/reporting/schema.yml

```diff
@@ -344,6 +344,16 @@ models:
 
   - name: reporting.vw_top_5_muni
     description: '{{ doc("view_vw_top_5_muni") }}'
+    columns:
+      - name: ccao_stage_used
+        data_tests:
+          - accepted_values:
+              name: reporting_vw_top_5_muni_ccao_stage_used_certified_in_prior_years
+              values:
+                - certified
+              config:
+                where: >
+                  CAST(year AS int) < {{ var('data_test_iasworld_year_start') }}
```

Comment:
Clearer regression test for the problem we saw with null AVs in `reporting.vw_top_5_muni`.